ddtrace/llmobs/_integrations/utils.py

```diff
@@ -878,12 +917,14 @@ def _openai_get_tool_definitions(tools: List[Any]) -> List[ToolDefinition]:
                 schema=_get_attr(custom_tool, "format", {}),  # format is a dict
             )
         # chat API function access and response API tool access
-        # only handles FunctionToolParam and CustomToolParam for response API for now
+        # only handles FunctionToolParam, CustomToolParam and McpListToolsTool for response API for now
```

Comment:
are we only handling attributes we know are present on FunctionTOolParam/CustomToolParam/McpListToolsTool or am I misunderstanding here what `only handles ...` means?